### PR TITLE
Feature/docker image

### PR DESCRIPTION
**Pull request by DaNautilus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -371,6 +371,50 @@ The last step is the easiest, just hit the following command in your terminal, b
...
+```shell
+docker build -t <your-image-name> .
+```
+
+### Run Docker image in container and map port
+
+#### Run image in detached mode
+
+```shell
+docker run -d -p <port-on-host>:<port-inside-docker-container> <your-image-name>
```
Comment (dweber019) [inline README.md]: Tell the user where they can find `port-inside-docker-container` the port => in .env

File: README.md [hunk 2]
```diff
@@ -371,6 +371,50 @@ The last step is the easiest, just hit the following command in your terminal, b
 npm start db.seed
 ```
 
+## Run in Docker container
```
Comment (dweber019) [inline README.md]: Add a section about how to configure the app settings. If a CI is deploying the project there isn't any `.env` file so we have to pass the env vars.

Comment (DaNautilus) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (DaNautilus) [reply README.md]: Done

**Pull request merged**